src/app/component/ubs/components/ubs-personal-information/ubs-personal-information.component.html

```diff
@@ -1,71 +1,131 @@
-      <div class="validation" *ngIf="lastName.invalid && lastName.touched">
-        <app-ubs-input-error [formElement]="lastName" [lastNameFieldValue]="lastName.value" [controlName]="'lastName'">
+      <label>{{'personal-info.info-surname' | translate}}</label>
+      <input
+        formControlName="lastName"
+        type="text"
+        class="shadow-none form-control">
+      <div
+        class="validation"
+        *ngIf="getControl('lastName').invalid && getControl('lastName').touched">
+        <app-ubs-input-error
+          [formElement]="getControl('lastName')"
+          [lastNameFieldValue]="getControl('lastName').value"
+          [controlName]="'lastName'">
         </app-ubs-input-error>
       </div>
     </div>
     <div class="col-12 col-sm-6 form-group">
       <label>Email</label>
-      <input formControlName="email" type="text" class="shadow-none form-control" placeholder="example@email.com">
-      <div class="validation" *ngIf="email.invalid && email.touched">
-        <app-ubs-input-error [formElement]="email" [emailFieldValue]="email.value" [controlName]="'email'"></app-ubs-input-error>
+      <input
+        formControlName="email"
+        type="text"
+        class="shadow-none form-control"
+        placeholder="example@email.com">
+      <div
+        class="validation"
+        *ngIf="getControl('email').invalid && getControl('email').touched">
+        <app-ubs-input-error
+          [formElement]="getControl('email')"
+          [emailFieldValue]="getControl('email').value"
+          [controlName]="'email'">
+        </app-ubs-input-error>
       </div>
     </div>
   </div>
   <hr>
   <div class="row adress-section">
     <div class="w-100">
-      <h2 class="address-title">Адреса вивозу сміття</h2>
+      <h2 class="address-title">{{'personal-info.info-address' | translate}}</h2>
+    </div>
+    <div *ngIf="!addresses; else addressesList" class="no-addresses">
+      <input formControlName="isAddress" style="display: none;">
+      <p>{{'personal-info.info-no-address' | translate}}</p>
     </div>
-    <div *ngIf="addresses.length === 0; else addressesList" class="no-addresses">У Вас немає доданих адрес. Додайте,
-      будь ласка, адресу.</div>
     <ng-template #addressesList>
-      <app-address *ngFor="let address of addresses" [address]="address" class="address col-12 col-md-6"></app-address>
+      <div class="address d-flex" *ngFor="let address of addresses">
+        <div class="custom-radio d-flex">
+          <input
+            type="radio"
+            [value]="address.id"
+            [checked]="address.checked"
+            (click)="checkAddress(address.id)">
+        </div>
+        <app-address [address]="address" class="address col-12 col-md-6">
+        </app-address>
+        <div class="edit-address" (click)="editAddress(address.id)">
+          <img src="../../../../../../assets/img/comments/edit.png" alt="">
+        </div>
```

Comment:
maybe here will be better to add binding to src and fill out "alt"?

Reply:
fixed